src/main/kotlin/no/nav/dagpenger/regel/api/RegelApi.kt

```diff
@@ -51,7 +54,7 @@ import java.util.concurrent.TimeUnit
 
 val APPLICATION_NAME = "dp-regel-api"
 
-fun main() {
+fun main() = runBlocking {
```

Comment:
Jeg må lære meg coroutines bedre, men dette betyr at at hovedtråden blokkerer? Har det noen implikasjoner for ktor?

Reply:
Dette betyr at main() ikke avslutter før andre coroutines fyrt opp i main scope er ferdig.